pennylane/_qubit_device.py

```diff
@@ -19,6 +19,7 @@
 # e.g. instead of expval(self, observable, wires, par) have expval(self, observable)
```

Comment:
The ``_wires_measured`` attribute can be removed as is from ``_qubit_device``: Lines 75, 88, 118 would not be relevant in ``execute()``

Reply:
This would affect ``test_qubit_device.py`` lines 288, 298

Reply:
done!